File: Python/number_guess.py
import random

print("Now I pick a number between 1 and 100")

a = 1
b = 100
target_number = random.randint(a,b)

guess_number = input("Now think of a number and put it here: ")
guess_number = int(guess_number)

no_attempts = 0

while (target_number!=guess_number):
    guessing_value = target_number - guess_number
    if(guessing_value >0 and guessing_value < 10):
        guess_number = input("You are bit low but close guess again: ")
        guess_number = int(guess_number)
        no_attempts += 1
    if(guessing_value >0 and guessing_value > 10):
        guess_number = input("you are too low guess again: ")
        guess_number = int(guess_number)
        no_attempts += 1
    if(guessing_value <0 and guessing_value < 10):
        guess_number = input("You are bit high but close guess again: ")
        guess_number = int(guess_number)
        no_attempts += 1
    if(guessing_value <0 and guessing_value > 10):
        guess_number = input("you are too high guess again: ")
        guess_number = int(guess_number)
        no_attempts += 1
print(f"you guessed correctly I guess {target_number} it took you {no_attempts} timess")

    

File: C#/number_guess.cs
using System;

class Guessing
{
    public static void Main(String[] args)
    {
        Random rnd = new Random();
        int target_value = rnd.Next(1, 100);
        Console.WriteLine("Welcome to the number guessing game!");
        Console.WriteLine("Guess the number between 1 and 100: ");
        String input = Console.ReadLine();
        int guess_value = 0;
        int.TryParse(input, out guess_value);
        int no_attempts = 0;

        while(target_value != guess_value)
        {
            int guess_factor = target_value - guess_value;

            if(guess_factor>0 && guess_factor>10)
            {
                Console.WriteLine("Wrong guessing, lower and far");
                Console.WriteLine("Guess again: ");
                input = Console.ReadLine();
                int.TryParse(input, out guess_value);
                no_attempts++;
            }
            if (guess_factor > 0 && guess_factor < 10)
            {
                Console.WriteLine("Wrong guessing, lower but close");
                Console.WriteLine("Guess again: ");
                input = Console.ReadLine();
                int.TryParse(input, out guess_value);
                no_attempts++;
            }
            if (guess_factor < 0 && guess_factor > 10)
            {
                Console.WriteLine("Wrong guessing, higher and far");
                Console.WriteLine("Guess again: ");
                input = Console.ReadLine();
                int.TryParse(input, out guess_value);
                no_attempts++;
            }
            if (guess_factor < 0 && guess_factor < 10)
            {
                Console.WriteLine("Wrong guessing, higher and close");
                Console.WriteLine("Guess again: ");
                input = Console.ReadLine();
                int.TryParse(input, out guess_value);
                no_attempts++;
            }
        }
        Console.WriteLine("Correct guessing it took you "+ no_attempts + "times");
    }
}